drake/solvers/NloptSolver.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DRAKE_NLOPT_SOLVER_H
```

Comment:
I continue to be confused as to the output you expect from cpplint.  am I not running it correctly?

drake-distro/drake/solvers% cpplint NloptSolver.h
Done processing NloptSolver.h
Total errors found: 0

I agree about the include guard and will fix.